app/models/crop.rb

```diff
@@ -206,7 +247,22 @@ def add_alternate_names_from_csv(alternate_names)
   # searches for crops whose names match the string given
   # just uses SQL LIKE for now, but can be made fancier later
   def self.search(query)
-    where("name ILIKE ?", "%#{query}%")
+    search_str = query.nil? ? "" : "#{query.downcase}"
+    response = __elasticsearch__.search(
+      {
+        query: {
+          multi_match: {
+            query: search_str,
+            fields: ["name", "scientific_names.scientific_name", "alternate_names.name"]
+          }
+        },
+        size: 50
+      }
+    )
+    response.records.to_a
```

Comment:
it might be just me, but i prefer to explicitly use the "return" keyword for lines like this, just so it's more readable